### PR TITLE
Adds a gamemode probability verb

### PR DESCRIPTION
**Pull request by printer16**
Using the show server revision verb will now show the percent chance of each gamemode in the secret and mixed secret rotation. Based off of how TG did it.

EDIT: added a new verb for it.

### CODE REVIEW

**Comment by Lohikar** [comment]
...why under `Show-Server-Revision`?

**Comment by printer16** [comment]
Where else?

**Comment by skull132** [comment]
A new verb?

**Comment by printer16** [comment]
Added a new verb for it.

File: code/controllers/configuration.dm [hunk 1]
```diff
@@ -273,6 +273,8 @@ var/list/gamemode_cache = list()
 
 	var/merchant_chance = 20 //Chance, in percentage, of the merchant job slot being open at round start
 
+	var/show_game_type_odd = 1 // If the game type odds are shown in server revision
```
Comment (Arrow768) [inline code/controllers/configuration.dm]: The description is misleading.
That var is used to change if the verb can be used or not

Comment (printer16) [inline code/controllers/configuration.dm]: Changed.

**Pull request merged**